client/app/veeam-enterprise/dashboard/license/veeam-enterprise-license.html

```diff
@@ -0,0 +1,34 @@
+<form novalidate name="licenseForm"
+    data-ng-submit="$ctrl.submitForm(licenseForm)">
+    <oui-modal data-heading="{{ ::($ctrl.action === 'register' ?
+        'veeam_enterprise_shortcut_label_activate_license':
+        'veeam_enterprise_shortcut_label_update_informations') | translate }}"
+        data-primary-label="{{ ::'common_modal_ok' | translate }}"
+        data-secondary-label="{{ ::'common_modal_cancel' | translate }}"
+        data-secondary-action="$ctrl.dismissModal()"
+        data-on-dismiss="$ctrl.dismissModal()"
+        data-loading="$ctrl.loading">
+        <oui-field data-label="{{ ::'veeam_enterprise_configuration_label_ip' | translate }}">
+            <input class="oui-input" type="text" name="ip"
+                data-ng-model="$ctrl.form.ip"
```

Comment:
Could have used AngularJS form validation instead

Reply:
It add an error `validIpAddress` in angularjs validation